### PR TITLE
Update enable-the-report-message-add-in.md

### PR DESCRIPTION
**Pull request by msbemba**
Cleared both (On prem and shared mailboxes) into different lines that way there is less ambiguity.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7921

File: microsoft-365/security/office-365-security/enable-the-report-message-add-in.md [hunk 1]
```diff
@@ -55,7 +55,9 @@ If you're a global administrator or an Exchange Online administrator, and Exchan
   - Outlook included with Microsoft 365 apps for Enterprise
   - Outlook app for iOS and Android
 
-- Both add-ins aren't available for shared mailboxes or mailboxes in on-premises Exchange organizations.
+- Both add-ins aren't available for shared mailboxes.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/enable-the-report-message-add-in.md]:
```suggestion
- Both add-ins are not available for shared mailboxes.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**